### ISSUE TITLE
Season Cap + Event Modifiers

### ISSUE THREAD

**Issue by specky532** [ruiaraujo012/fx-clash-stats-web#48]
Would be great to have a season cap selector to build my best team within certain series restrictors. Ie I have lots of season 8 geat but can only use up to season 6 gear in my tournament, would be great to select what series max to use.

Also to have an ability to add the 10% boost that certain drivers get during certain events.

**Comment by ruiaraujo012** [comment]
True, that would be a good improvement. Currently I'm out of time unfortunately 😕

**Comment by ruiaraujo012** [comment]
To the 10% boost I have an issue already #21
